### ISSUE TITLE
Improve the mobile editor's accessibility

### ISSUE THREAD

**Issue by dcalhoun** [WordPress/gutenberg#54362]
## What problem does this address?

There are portions of the mobile editor that are less accessible. 

## What is your proposed solution?

We should make strives to address blocking issues and improve accessibility overall. 

- [ ] Evaluate current screen reader flows capturing individual improvement opportunities as separate issues. E.g. text entry, block navigation, focus management, dynamic content change announcements.[^1] 
- [ ] Evaluate current support for a11y features like dynamic text site, increased contrast ratio, reduced motion, etc.[^1] 
- [ ] Review and address issues included in the [Google Play pre-launch report](https://developer.android.com/guide/topics/ui/accessibility/testing#pre-launch-report). 

[^1]: Any findings or improvement opportunities should be captured as separate issues. Keep in mind existing issues may already exist in [`gutenberg`](https://github.com/WordPress/gutenberg/issues?q=is%3Aopen+is%3Aissue+label%3A%22Mobile+App+-+i.e.+Android+or+iOS%22+label%3A%22%5BType%5D+Accessibility+%28a11y%29%22) or [`gutenberg-mobile`](https://github.com/wordpress-mobile/gutenberg-mobile/issues?q=is%3Aopen+is%3Aissue+label%3AAccessibility). If so, those should be re-prioritized and deduplicated as necessary.

**Comment by tyxla** [comment]
This is no longer relevant after the React Native implementation was removed in #78747. Closing.
